src/views/common.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components'
+
+export const Page = styled.div`
+  width: 100%;
+`
+
+export const Content = styled.div`
+	width: auto;
+			
+	display: flex;
+	flex-direction: column;
+	gap: 2rem;
+
+	margin: 2rem 5rem;
+
+	color: #514E4E;
+
+	& > p {
+		font-size: 1.25rem;
```

Comment:
A `font-size` do parágrafo pode ser 1rem mesmo

```suggestion
		font-size: 1rem;
```